ev3dev2/console.py

```diff
@@ -38,7 +38,62 @@ def __init__(self, font="Lat15-TerminusBold24x12"):
 
         """
         self._font = None
-        self.set_font(font, False)  # don't reset the screen during construction
+        self._columns = 0
+        self._rows = 0
+        self._echo = False
+        self._cursor = False
+        self.set_font(font, reset_console=False)  # don't reset the screen during construction
+        self.cursor = False
```

Comment:
Is there a reason to set both `_cursor` and `cursor`?

Reply:
My linter says that class-level fields should be instantiated/initialized in the constructor. Agree that it's redundant.